mindmeld/path.py

```diff
@@ -108,23 +108,25 @@
 DUCKLING_UBUNTU18_PATH = os.path.join(
     RESOURCES_FOLDER, "duckling-x86_64-linux-ubuntu-18"
 )
-DUCKLING_CENTOS8_PATH = os.path.join(RESOURCES_FOLDER, "duckling-x86_64-centos-8-core")
+DUCKLING_UBUNTU20_PATH = os.path.join(
+    RESOURCES_FOLDER, "duckling-x86_64-linux-ubuntu-20"
+)
 DUCKLING_OSX_PATH = os.path.join(RESOURCES_FOLDER, "duckling-x86_64-osx")
-DUCKLING_UBUNTU16_MD5 = "1e58b4e91d580d98c8ac4f5a69b0ebd1"
-DUCKLING_UBUNTU18_MD5 = "b9c4891b27731df97d7f01c33441a91c"
-DUCKLING_OSX_MD5 = "d01753261e6e7940b533f09dc17d0c19"
-DUCKLING_CENTOS8_MD5 = "f840176c2b96c0037edd9524faac5e93"
+DUCKLING_UBUNTU16_MD5 = "f32c9c1c8e83526dac430f4a25bdcf80"
+DUCKLING_UBUNTU18_MD5 = "843cc934c41585a875c32bd3f2913df0"
+DUCKLING_UBUNTU20_MD5 = "281aff210f4c0a4d8661f8a1f0aabc13"
+DUCKLING_OSX_MD5 = "ef80f9348a500a0b06cd0de85a414e90"
```

Comment:
Can we change these to SHA256 checksums?  MD5 is pretty badly broken at this point.

Reply:
done